### ISSUE TITLE
Feature request: ignore any children whose values are false or null

### ISSUE THREAD

**Issue by antonivlev** [cedricdelpoux/react-responsive-masonry#42]
Hi! Thank you for this library.

When using the `<Masonry>` component, sometimes we conditionally hide the children. This however messes up the layout, see https://codesandbox.io/s/exciting-chandrasekhar-v3zq5?file=/src/App.js

We have work arounds, but would it be possible to completely ignore a child (so don't wrap it with a container) if it's falsy?

**Comment by RavdeepSingh99** [comment]
@cedricdelpoux Hi, how are you?

I'd face the same issue as mentioned by @antonivlev.

I am able to fix this issue, as the issue is in `Masonry Component`. Please Let me know if you are accepting PR on this issue.
Thanks!

**Comment by RavdeepSingh99** [comment]
@antonivlev A temporary solution is that you can add this SCSS to Masonry Component. This SCSS can be used for the codesandbox that you shared, which is first element in first column being falsy. Downside to this method is that column and element should be known in advance.
```
masonry: {
  '& > div:nth-of-type(1)': {
    '& > div:nth-of-type(1)': {
        marginTop: '0px !important'
    }
  }
}
```

**Comment by cedricdelpoux** [comment]
published in `2.1.3`
